ste/sender-azureFile.go

```diff
@@ -531,39 +531,28 @@ func (d AzureFileParentDirCreator) CreateParentDirToRoot(ctx context.Context, fi
 }
 
 func (d AzureFileParentDirCreator) CreateDirToRoot(ctx context.Context, shareClient *share.Client, directoryClient *directory.Client, t FolderCreationTracker) error {
```

Comment:
any way we can make useful utilities like this more testable?

Reply:
These are tested via e2e framework, we use them to create resources